amgut/db/0042.sql

```diff
@@ -0,0 +1,129 @@
+CREATE TABLE ag."source" (
+	source_id            uuid DEFAULT ag.uuid_generate_v4() ,
+	source_name          varchar(100)  ,
+	ag_login_id          uuid  ,
```

Comment:
That is only creating a FK for the column barcode not for the column ag_login_id...